apps/molgenis-components/src/components/forms/InputRangeLong.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <FormGroup
+    :id="id + '-from'"
+    :label="label"
+    :description="description"
+    :errorMessage="errorMessage || bigIntError1 || bigIntError2"
+  >
+    <InputGroup class="d-flex">
+      <template v-slot:prepend>
+        <slot name="prepend"></slot>
+      </template>
+      <BaseInputLong
+        :id="id + '-from'"
+        :modelValue="modelValue[0]"
+        @update:modelValue="emitValue($event, 0)"
+        placeholder="from"
+        :class="{ 'is-invalid': errorMessage || bigIntError1 }"
+      />
+      <BaseInputLong
+        :id="id + '-to'"
+        :modelValue="modelValue[1]"
+        @update:modelValue="emitValue($event, 1)"
+        placeholder="to"
+        :class="{ 'is-invalid': errorMessage || bigIntError2 }"
+      />
+      <template v-slot:append>
+        <slot name="append"></slot>
+      </template>
+    </InputGroup>
+  </FormGroup>
+</template>
+
+<script>
+import BaseInput from "./baseInputs/BaseInput.vue";
+import BaseInputLong from "./baseInputs/BaseInputLong.vue";
+import FormGroup from "./FormGroup.vue";
+import InputGroup from "./InputGroup.vue";
+import {getBigIntError} from '../utils';
```

Comment:
I've noticed that in a lot of these files, the layout is not consistent with our settings. Maybe your edit uses slightly different settings for Vue3? Would be nice if it'd stay the same.

Reply:
Sorry about the formatting. Meanwhile hopefully fixed the TableExplorer error, thx